lib/components/Autosuggest/Autosuggest.treat.ts

```diff
@@ -39,3 +39,11 @@ export const menu = [menuBase, menuHeight];
 export const groupHeading = style({
   textTransform: 'uppercase',
 });
+
+export const input = style({
+  selectors: {
+    [`&::-webkit-search-cancel-button`]: {
```

Comment:
This should probably be part of the input reset. That way all fields will remove this behaviour.
https://github.com/seek-oss/braid-design-system/blob/a6ef88d129caa679ffce6719fbf944dfed2b46a5/lib/reset/reset.treat.ts#L67

Reply:
Good suggestion. I've made that change.